app/models/step_by_step_page.rb

```diff
@@ -90,6 +90,17 @@ def discard_notes
     internal_change_notes.where(edition_number: nil).delete_all
   end
 
+  def issues
```

Comment:
Is this the same as the `issues` method in `DateParser`? Why are both needed?

Reply:
Have now removed 👍